Dockerfile

```diff
@@ -4,19 +4,27 @@ FROM debian:stable-slim AS mirrorbits
 ## and the risk of a breaking behavior is evaluated as low
 # hadolint ignore=DL3008
 RUN apt-get update && \
-  apt-get install --no-install-recommends -y tar curl ca-certificates && \
+  apt-get install --no-install-recommends -y tar curl ca-certificates git && \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+ARG go_version=1.21.3
```

Comment:
- We should use go 1.11 (or 1.10) as per their travis file for the `0.5.1` version: https://github.com/etix/mirrorbits/blob/v0.5.1/.travis.yml#L9
- We should change the base image on this "Dockerfile stage" to use golang instead of debian (but keep debian for the final image of course)